packages/block-editor/src/utils/get-font-styles-and-weights.js

```diff
@@ -79,7 +79,10 @@ export function getFontStylesAndWeights( fontFamilyFaces ) {
 
 	fontFamilyFaces?.forEach( ( face ) => {
 		// Check for variable font by looking for a space in the font weight value. e.g. "100 900"
-		if ( /\s/.test( face.fontWeight.trim() ) ) {
+		if (
+			'string' === typeof face.fontWeight &&
+			/\s/.test( face.fontWeight.trim() )
```

Comment:
If we're guarding against extra whitespace for space-separated values, I'm wondering if we should trim all `face.fontWeight` string values, e.g., 

```js
		face.fontWeight =
			'string' === typeof face.fontWeight
				? face.fontWeight.trim()
				: face.fontWeight;
		if ( /\s/.test( face.fontWeight ) ) {
```

Reply:
Ah yes, true. Thanks for the explanation 👍🏻 

> I thought about writing tests for it right after adding the solution but then decided to put in the effort after we agreed on the solution's approach.

Maybe tests will reveal some optimizations too. For now, it LGTM to me since it fixes a pretty major bug 🚀